llvm/docs/GitHub.rst

```diff
@@ -214,6 +214,13 @@ commonly used first:
   request will understand that you're rebasing just your patches, and display
   this result correctly with a note that a force push did occur.
 
+.. note::
+   When using `Squash and merge` in LLVM’s GitHub web interface, GitHub will
+   use the pull request title and description as the commit message. Only
+   users with commit access can edit the commit message just before merging a PR.
```

Comment:
Split this sentence out from the explanation of what does and does not get used.

Then you can say:
"Only users with commit access are able to merge PRs, and while doing so, they will be given one last chance to edit the commit title and message.

If you do not have commit access, make sure that your PR title and description are up to date before...."

Reply:
This whole timing issue is a pain that I am trying to address elsewhere. I hope to get reviewers into the habit of recognising when the details are out of sync at least.